test/e2e/connectivity_test.go

```diff
@@ -192,7 +192,12 @@ func TestPodConnectivityDifferentNodes(t *testing.T) {
 
 func (data *TestData) redeployAntrea(t *testing.T, enableIPSec bool) {
 	var err error
-
+	// export logs before deleting Antrea
+	if enableIPSec {
+		exportLogs(t, data, "beforeRedeployAntreaWithIPSecEnabled", false)
+	} else {
+		exportLogs(t, data, "beforeRedploy", false)
```

Comment:
nit: I feel like for symmetry this should be "beforeRedployAntrea" or "Antrea" should be removed from "beforeRedeployAntreaWithIPSecEnabled". I recommend the latter to keep the directory name shorter. "beforeRedeployWithIPsec" would be best IMO.

Reply:
Yes actually it's shorter and symmetric, thanks. Updated the PR.